_includes/install/file-system-perms-oneuser.md

```diff
@@ -22,7 +22,7 @@ To set permissions before you install the Magento software:
 3.	If you haven't done so already, get the Magento software in one of the following ways:
 
 	*	[Compressed archive]({{ page.baseurl }}/install-gde/prereq/zip_install.html)
-	*	[Composer metapackage]({{ page.baseurl }}/install-gde/prereq/integrator_install.html)
+	*	[Composer metapackage]({{ page.baseurl }}/install-gde/composer.html)
```

Comment:
This include is also heavily used in 2.0. Might need to add conditional for version, build another include, other option?

Reply:
I don't think the changes I made require special handling for 2.0. I think they apply well to all versions.